src/test/groovy/io/openliberty/tools/gradle/DevTest.groovy

```diff
@@ -181,14 +180,11 @@ class DevTest extends BaseDevTest {
 
     @Test
     public void generateFeatureTest() throws Exception {
-        final String GENERATED_FEATURES_FILE_NAME = "generated-features.xml";
-        final String SERVER_XML_COMMENT = "Plugin has generated Liberty features"; // the explanation added to server.xml
-        final String NEW_FILE_INFO_MESSAGE = "This file was generated by the Liberty Gradle Plugin and will be overwritten"; // the explanation added to the generated features file
-
+        assertFalse(verifyLogMessage(10000, "batch-1.0", errFile)); // not present on server yet
         // Verify generate features runs when dev mode first starts
         assertTrue(verifyLogMessage(10000, RUNNING_GENERATE_FEATURES));
-        int generateFeaturesCount = countOccurrences(RUNNING_GENERATE_FEATURES, logFile);
-        assertFalse(verifyLogMessage(10000, "batch-1.0", errFile)); // not present on server yet
```

Comment:
Why did we remove this line? Should we remove in ci.maven as well? https://github.com/OpenLiberty/ci.maven/pull/1540/files#diff-94179a656c021421e9e696ddf72cf24f58d8247ef199afdfdab55895d18c0f90R267

Reply:
It just moved up to line 183 in the new file.